README.md

```diff
@@ -8,25 +8,25 @@
 
 ### Features
 
-- Integrates [detekt](https://github.com/arturbosch/detekt) for code analysis
-- Default quality profile `detekt active` (80 rules) and `detekt all` (164 rules)
-- Syntax highlighting
-- Supports SonarQube 6.7.7 and up
-- Supports detekt's `yaml config` and `baseline.xml` (also `path filters`)
-- Jacoco support
-- Lines of code and complexity metrics
+- Integrates [detekt](https://github.com/arturbosch/detekt) for static code analysis
+- Default quality profiles `detekt active` (80+ rules) and `detekt all` (164+ rules)
+- Supports SonarQube 7.9.3+
+- Supports detekt's `yaml config`, `baseline.xml` and `excludes`
+- Seamless integration with official SonarKotlin (no redundant features)
```

Comment:
Maybe the `README.md` should also mention explicitly what's not supported. Otherwise, this could end in bug reports in the form of GitHub issues.